### DISCUSSION TITLE
Toggle column visible

### DISCUSSION THREAD

**Discussion by pgck** [nick-keller/react-datasheet-grid#98] [Q&A] [answered]
First of all... this component is awesome, but we've found an issue with changing the column visibility.
Updating the columns array seems to be not noticed, and columns doesn't update.
Working example can be found here: https://codesandbox.io/s/compassionate-noyce-w78rt?file=/src/App.js

Is there a way to update columns visibility?

**Comment by nick-keller** [comment, marked as answer]
Hello @pgck ,
Yes and it is very straight forward, just use the dynamic version: https://react-datasheet-grid.netlify.app/docs/performance/static-vs-dynamic
